src/main/java/uk/gov/ons/ssdc/supporttool/model/entity/Survey.java

```diff
@@ -39,4 +39,7 @@
 
   @OneToMany(mappedBy = "survey")
   private List<FulfilmentSurveyPrintTemplate> fulfilmentPrintTemplates;
+
+  @OneToMany(mappedBy = "survey")
+  private List<FulfilmentSurveySmsTemplate> smsTemplates;
```

Comment:
this change needs to be mirrored in Case Processor, Case API, Response Operations (& Notify Service?) and DDL?

Reply:
Made the changes everywhere. No ddl change needed

Reply:
There should be a foreign key constraint in the `fulfilment_survey_sms_template` table to ensure that the `survey_id` column is constrained to only ID values in the `survey` table

Reply:
try running the DDL generator on the case processor branch... I bet it will create a FK constraint

Reply:
I did try but it came up with no changes 🤔 Probably my mistake so let me try again